datajoint/expression.py

```diff
@@ -101,14 +104,13 @@ def where_clause(self):
         return '' if not self.restriction else ' WHERE(%s)' % ')AND('.join(
             str(s) for s in self.restriction)
 
-    def make_sql(self, fields=None):
+    def make_sql(self, fields=None, distinct=True):
```

Comment:
Hmm, don't think this is serving any purpose now, right?

```suggestion
    def make_sql(self, fields=None):
```